scripts/updates/utils/termux_pkg_upgrade_version.sh

```diff
@@ -96,6 +96,13 @@ termux_pkg_upgrade_version() {
 		fi
 	done
 
+	# Get available space in 1024-byte blocks (df outputs sizes in 1024-byte blocks)
+	local space_available="$(df -P "/var/lib/docker" | awk 'NR==2 {print $4}')"
+
+	# Determine if there is at least 2 GiB available.
+	# 2 GiB = 2 * 1024^2 blocks (since each block is 1024 bytes)
+	local enough_space=$(( space_available >= 2 * 1024 ** 2 ))
```

Comment:
```suggestion
```
This feels like an odd abstraction to make.
Just put it in the condition below.

Also this is Mebibytes, not Gibibytes.

Reply:
Wait a minute, no you were right I forgot `df` reports in 1KiB blocks.